src/main/java/net/snowflake/client/log/JDK14JCLWrapper.java

```diff
@@ -16,6 +16,10 @@ public JDK14JCLWrapper(String className) {
     this.logger = new JDK14Logger(className);
   }
 
+  public SFLogger getLogger() {
```

Comment:
can this be non-public but a package private if only for tests?

Reply:
True! Changed.